src/DatePicker/DatePicker.js

```diff
@@ -9,14 +9,16 @@ import React, { Component } from 'react';
 class DatePicker extends Component {
     constructor(props) {
         super(props);
-
+        const ISO_DATE_FORMAT = 'YYYY-MM-DD';
+        let formattedDate = props.value.length > 0 ? moment(props.value, ISO_DATE_FORMAT).format(this.getLocaleDateFormat()) : '';
         this.state = {
             hidden: true,
-            selectedDate: null,
+            selectedDate: formattedDate.length === 0 ? null : moment(formattedDate, this.getLocaleDateFormat()),
             arrSelectedDates: [],
-            formattedDate: ''
+            formattedDate
         };
 
+        this.getLocaleDateFormat = this.getLocaleDateFormat.bind(this);
```

Comment:
We shouldn't need to bind with an arrow function

Reply:
Hey, I just pushed a change removing that line.